### ISSUE TITLE
Create project from json export

### ISSUE THREAD

**Issue by Felixmil** [esqLABS/esqlabsR#717]

**Comment by PavelBal** [comment]
@Felixmil implemented?

**Comment by Felixmil** [comment]
No, these are not the same json as the snapshot I implemented. I need to get back to this and rework it with latest change. When I'm done with project work, maybe one day 🙈

**Comment by Felixmil** [comment]
Implemented. `loadProject(path)` reads a `Project.json` container plus its `definitions/` tree (and still accepts a single inlined `Project.json`), and `restoreProject(snapshot, dir)` materializes a `.esqlabsR` snapshot back into a full project folder, returning a freshly loaded `Project`.

Closing as done. Design record: #1088. Consolidation PR: #1017.
